qa/suites/crimson-rados/thrash_simple/objectstore/seastore/seastore-rbm.yaml

```diff
@@ -0,0 +1,21 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        # crimson's osd objectstore option
+        crimson osd objectstore: seastore
+        seastore main device type: RANDOM_BLOCK_SSD
+        debug seastore: 20
+        debug seastore onode: 20
+        debug seastore odata: 20
+        debug seastore omap: 20
+        debug seastore tm: 20
+        debug seastore t: 20
+        debug seastore cleaner: 20
+        debug seastore epm: 20
+        debug seastore lba: 20
+        debug seastore fixedkv tree: 20
+        debug seastore cache: 20
+        debug seastore journal: 20
+        debug seastore device: 20
+        debug seastore backref: 20
```

Comment:
I would prefer introducing the new seastore variant in a common place and symlink it here so we can reuse later on. Do we want to begin with this variant in thrash_simple only for now? If so, we can do that later.

Let's add a `$` file (or rename to `objectstore/seastore$/`) so that each time we run thrash_simple either seastore or seastore-rbm will be picked randomly and not both. What do you think?

Reply:
> I would prefer introducing the new seastore variant in a common place and symlink it here so we can reuse later on. Do we want to begin with this variant in thrash_simple only for now? If so, we can do that later.
> 
> Let's add a `$` file (or rename to `objectstore/seastore$/`) so that each time we run thrash_simple either seastore or seastore-rbm will be picked randomly and not both. What do you think?

Thanks for your comment. I'll address it.

Reply:
> Let's add a $ file (or rename to objectstore/seastore$/) so that each time we run thrash_simple either seastore or seastore-rbm will be picked randomly and not both.

In this case, how to schedule tests that only pick seastore not randomly?

Reply:
The random picks would be between the Seastore variants only. `--filter seastore` would still work.

Reply:
@Matan-B @myoungwon it doesn't look like `--filter seastore-rbm` can force the all tests to select seastore-rbm, I've tried:
```
teuthology-suite -m smithi -s crimson-rados -c wip-qa-seastore-rbm-test --filter seastore-rbm --dry-run
```
Looks the number of scheduled tests isn't a constant, suggesting teuthology firstly pick seastore/seastore-rbm randomly, then filter out the rbm ones.

We might need to be able to run in both ways:
* select seastore/seastore-rbm randomly for all the target test cases.
* force seastore or seastore-rbm for all the target test cases.